examples/ui.apps/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-site/scripts/search.js

```diff
@@ -61,23 +61,37 @@
             }
         });
 
-        if (noResults) {
-            noResults.style.display = (total === count) ? "" : "none";
+        updateSearchResultsInfoMessage(token, foundComponentTeasers);
+    }
+
+    // useful for Accessibility, helping users with low vision and users with cognitive disabilities to identify the change in results
+    function updateSearchResultsInfoMessage(token, foundComponentTeasers) {
+        if (!token) {
+            document.querySelector(selectors.searchResultsInfoMessage).innerHTML=''
+        } else {
+            document.querySelector(selectors.searchResultsInfoMessage).innerHTML = foundComponentTeasers > 0 ?
+                `<p aria-live="polite" role="status">${foundComponentTeasers} ${foundComponentTeasers === 1 ? "component" : "components"} found based on your search.</p>` :
+                `<p aria-live="polite" role="status">No components found. Please try a different search.</p>`;
         }
     }
 
+    function createSearchResultsInfoElement() {
+        var searchInput = document.querySelector(selectors.self);
+        var infoMessageContainer = document.createElement("div");
+        infoMessageContainer.setAttribute("id", "searchResultsInfoMessage");
+        infoMessageContainer.setAttribute("aria-live", "polite");
```

Comment:
Is it necessary to put `aria-live="polite"` on both parent `div` and `p` element?